### ISSUE TITLE
Issue rendering mathematics in boxes

### ISSUE THREAD

**Issue by hexylena** [galaxyproject/training-material#3951]
I have encountered a problem with rendering this mathematical expression. It occurs when used in a line block inside the details block of the hands-on section. The math expression is displayed before the details block is expanded.

Collapsed details block:
![collapsed](https://user-images.githubusercontent.com/15349569/225601131-2dd94103-551a-483b-ad47-ef188b36311d.png)

Expanded details block:
![expanded](https://user-images.githubusercontent.com/15349569/225601147-94933ba7-ee84-4af8-9ffb-59aa204f89ca.png)

I tried to find something similar in previous materials, it seems like [here](https://github.com/galaxyproject/training-material/blob/d13771e996e13fdc821a28556692a8e4e1675291/topics/single-cell/tutorials/scrna-raceid/tutorial.md?plain=1#L429) someone had the same issue. But there, it is [rendering fine](https://training.galaxyproject.org/training-material/topics/single-cell/tutorials/scrna-raceid/tutorial.html#performing-the-clustering), so maybe this only occurs when running the GTN page in Gitpod?

_Originally posted by @xtrojak in https://github.com/galaxyproject/training-material/pull/3949#discussion_r1138497091_

**Comment by xtrojak** [comment]
@hexylena any progress on this? The equation in the recently merged [tutorial](https://training.galaxyproject.org/training-material/topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.html?utm_source=matrix&utm_medium=newsbot&utm_campaign=matrix-news#retention-index-calculation) is not rendering correctly.

**Comment by hexylena** [comment]
no, no progress since march (smorgasbord preparation, other higher priorities have precluded investigating this issue.)

I understand it doesn't render perfectly, but it does render though, right? We have the equation rendered (when the box is closed as well as open) and then an (admittedly ugly) text version of the equation, so, it's rendering the required information sufficiently that someone can follow along, right?

**Comment by xtrojak** [comment]
Yeah, sure, it's functional so no worries.
